frontend/src/store/students.js

```diff
@@ -26,6 +26,24 @@ export const studentActions = {
         return response     
     },
 
+    // PATCH Student Data to backend 
+    async patchStudent({dispatch, commit}, studentData){
+        const patchStudent = {
+            method: "PATCH",
+            headers: {
+              'Accept': 'application/json',
+              'Content-Type': 'application/json'
+            },
+            body: JSON.stringify(
+              studentData 
+            )
+          }
+        const response = await fetch(`/api/students/${studentData.StudentID}`, patchStudent)
+        if(response.status === 200){
+            dispatch("getAllStudents")
+            commit('SET_UPDATE_STUDENT_SUCCESS', true)
+        }
```

Comment:
```suggestion
        if(response.status === 200){
            dispatch("getAllStudents")
            commit('SET_UPDATE_STUDENT_SUCCESS', true)
        } else {
            commit('SET_UPDATE_STUDENT_SUCCESS', false) 
        }
```

We should set the `SET_UPDATE_STUDENT_SUCCESS` to `false`, if the `PATCH` fails